src/coreclr/jit/flowgraph.cpp

```diff
@@ -3601,13 +3592,13 @@ PhaseStatus Compiler::fgCreateThrowHelperBlocks()
     assert(!fgRngChkThrowAdded);
 
     const static BBjumpKinds jumpKinds[] = {
-        BBJ_NONE,  // SCK_NONE
-        BBJ_THROW, // SCK_RNGCHK_FAIL
-        BBJ_THROW, // SCK_DIV_BY_ZERO
-        BBJ_THROW, // SCK_ARITH_EXCP, SCK_OVERFLOW
-        BBJ_THROW, // SCK_ARG_EXCPN
-        BBJ_THROW, // SCK_ARG_RNG_EXCPN
-        BBJ_THROW, // SCK_FAIL_FAST
+        BBJ_ALWAYS, // SCK_NONE
```

Comment:
This seems odd. Does SCK_NONE ever get used?

Reply:
I guess not; I added an assert in to test if `add->acdKind == SCK_NONE` is ever true, and the SuperPMI replay was clean. A quick search of the source code doesn't yield any places where we assign `SCK_NONE`, so maybe I can add an assert in here that assures we don't use `SCK_NONE`? Then I can remove the conditional logic below for setting the jump target if `newBlk` is a `BBJ_ALWAYS`.